.clang-tidy

```diff
@@ -31,7 +31,6 @@ modernize-*,\
 -modernize-return-braced-init-list,\
 -modernize-use-auto,\
 -modernize-use-trailing-return-type,\
--modernize-use-emplace,\
```

Comment:
Why remove this after you apply it?

Reply:
Because all such warnings are now fixed.

Reply:
But they might re-appear in the future? As development continues?

Reply:
The `-` sign indicates it was a suppression of a warning type, and now it's not suppressed.

Reply:
Oh, that's what I was missing 🤦

Reply:
Yeah, the syntax is a sorta weird mix in inclusions and exclusions: https://clang.llvm.org/extra/clang-tidy/